pkgs/development/php-packages/redis/default.nix

```
@@ -1,10 +1,14 @@
-{ buildPecl, lib, php }:
-
-buildPecl {
+{ buildPecl, lib, php, fetchFromGitHub }:
+buildPecl rec {
```

```suggestion
{ buildPecl, lib, php, fetchFromGitHub }:

buildPecl rec {
```

Please keep the empty line here.